### ISSUE TITLE
 When TaskRun is applied of tutorial, the error `json: unknown field "resource"` is output

### ISSUE THREAD

**Issue by masanetes** [tektoncd/pipeline#2249]
# Expected Behavior

According to `Running your Task` section of tutorial. TaskRun has a `spec.resouce` field and it seems to be successful.

# Actual Behavior

```
$ k apply -f something-task-run.yaml

Error from server (BadRequest): error when creating "something-task-run.yaml": admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "resource"
```

# Steps to Reproduce the Problem

1. Apply TaskRun according to the tutorial
https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md

# Additional Info

- Kubernetes version:

```
$ k version

Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-14T04:24:34Z", GoVersion:"go1.12.13", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.5", GitCommit:"20c265fef0741dd71a66480e35bd69f18351daea", GitTreeState:"clean", BuildDate:"2019-10-15T19:07:57Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
  ```

- Tekton Pipeline version:

```
$ tkn version

Client version: 0.8.0
Pipeline version: v0.11.0-rc2
```

As far as I can see the type TaskRunSpec has not `resource` field. but modified `resource` to `resources` then successful it.

https://github.com/tektoncd/pipeline/blob/1fb9a826dc83ff2a4864269c5051a2ac6d022e6b/pkg/apis/pipeline/v1beta1/taskrun_types.go#L31

Is this a simple tutorial readme mistake?

**Comment by vdemeester** [comment]
@masanetes thanks for the issue, it is fixed by #2248, and indeed a type in the tutorial :man_facepalming:

**Comment by masanetes** [comment]
@vdemeester ok, thanks!!
